src/prop/sat_solver.h

```diff
@@ -170,8 +170,23 @@ class CDCLTSatSolver : public SatSolver
    */
   virtual std::vector<Node> getOrderHeap() const = 0;
 
+  /**
+   * Get proof, which is used if prop-proof-mode is PROOF.
```

Comment:
I assume `prop-proof-mode` is a configuration option? Maybe use `option prop-proof-mode` instead?